misc/testerina/modules/testerina-core/src/main/ballerina/Module.md

```diff
@@ -214,79 +317,113 @@ A custom mock object can be defined in place of the real object which should con
 
 _main.bal_
 ```ballerina
-http:Client clientEndpoint = new("http://petstore.com");
+import ballerina/http;
+
+http:Client clientEndpoint = check new ("http://petstore.com");
 
-function getPet(string petId) returns Pet | error {
-  http:Response|error result = clientEndpoint->get("/pets?id="+petId);
-  if(result is error) {
-    return result;
-  } else {
-      Pet pet = constructPetObj(result); 
-    return pet;
-  }
+function getPet(string petId) returns http:Response | error {
+  http:Response result = check clientEndpoint->get("/pets?id="+petId);
+  return result;
+}
+
+function deletePet(string petId) returns error? {
+    http:Response res = check clientEndpoint->delete("/pets/delete?id="+petId);
 }
 ```
 
 _test.bal_
 ```ballerina
+import ballerina/test;
+import ballerina/http;
+
 // Mock object definition for http:Client object
-public type MockHttpClient client object {
-    public remote function get(@untainted string path, public 
-        http:RequestMessage message = ()) returns http:Response|http:ClientError {
-        http:Response res = new;
-        res.statusCode = 500;
-        return res;
+public client class MockHttpClient {
+    remote isolated function get(@untainted string path, map<string|string[]>? headers = (),
+    http:TargetType targetType = http:Response) returns http:Response | http:PayloadType | http:ClientError {
+        http:Response response = new;
+        response.statusCode = 500;
+        return response;
     }
-};
+}
 
 @test:Config {}
-function testGetPet() {
+function testTestDouble() returns error? {
     //mock object that would act as the test double to the clientEndpoint
-    clientEndpoint = <http:Client>mock(http:Client, new MockHttpClient());
-    http:Response res = getPet("D123");
+    clientEndpoint = test:mock(http:Client, new MockHttpClient());
+    http:Response res = check getPet("D123");
     test:assertEquals(res.statusCode, 500);
 }
 ```
 
 #### Stubbing member functions and variables of an object
 
-The member functions and variables are stubbed to return a specific value or to do nothing when invoked. Using the test module, a default mock object of the specified type. The default action of any member function/variable is to panic upon invocation/retrieval.  Subsequent to mock object creation, the required functions and variables of the default mock object should be stubbed to return a value or to do nothing when called.
+The member functions and variables are stubbed to return a specific value or to do nothing when invoked. Using the test module, a default mock object of the specified type. The default action of any member function/variable is to panic upon invocation/retrieval.  After mock object creation, the required functions and variables of the default mock object should be stubbed to return a value or to do nothing when called.
```

Comment:
```suggestion
The member functions and variables are stubbed to return a specific value or to do nothing when invoked. Using the test module, a default mock object of the specified type. The default action of any member function/variable is to panic upon invocation/retrieval.  After mock object creation, the required functions and variables of the default mock object should be stubbed to return a value or to do nothing when called.
```
This sentence is not complete: "Using the test module, a default mock object of the specified type."